build.sbt

```diff
@@ -9,27 +9,36 @@ ThisBuild / publishFullName := "Ross A. Baker"
 ThisBuild / githubWorkflowTargetBranches := List("*", "series/*")
 ThisBuild / githubWorkflowPublishTargetBranches := Seq(RefPredicate.StartsWith(Ref.Tag("v")))
 
-val JawnVersion             = "1.1.2"
-val Fs2Version              = "3.0.6"
-val MunitVersion            = "0.7.27"
-val MunitCatsEffectVersion  = "1.0.5"
+val JawnVersion = "1.2.0"
+val Fs2Version = "3.0.6"
+val MunitVersion = "0.7.27"
+val MunitCatsEffectVersion = "1.0.5"
 
-enablePlugins(SonatypeCiReleasePlugin)
+lazy val root = project
+  .in(file("."))
+  .enablePlugins(NoPublishPlugin)
+  .aggregate(`jawn-fs2`.jvm, `jawn-fs2`.js)
 
-libraryDependencies ++= Seq(
-  "org.typelevel"  %% "jawn-parser"           % JawnVersion,
-  "co.fs2"         %% "fs2-core"              % Fs2Version,
-  "org.typelevel"  %% "jawn-ast"              % JawnVersion             % Test,
-  "co.fs2"         %% "fs2-io"                % Fs2Version              % Test,
-  "org.scalameta"  %% "munit"                 % MunitVersion            % Test,
-  "org.typelevel"  %% "munit-cats-effect-3"   % MunitCatsEffectVersion  % Test
-)
-
-versionIntroduced := Map(
-  "3.0.0-M1" -> "1.0.1",
-  "3.0.0-M2" -> "1.0.1",
-  "3.0.0-M3" -> "1.0.1",
-  "3.0.0-RC1" -> "1.1.0",
-  "3.0.0-RC2" -> "1.1.1",
-  "3.0.0-RC3" -> "1.1.2"
-)
+lazy val `jawn-fs2` = crossProject(JVMPlatform, JSPlatform)
+  .crossType(CrossType.Full)
+  .in(file("."))
+  .enablePlugins(SonatypeCiReleasePlugin)
+  .settings(
+    libraryDependencies ++= Seq(
+      "org.typelevel" %%% "jawn-parser" % JawnVersion,
+      "co.fs2" %%% "fs2-core" % Fs2Version,
+      "org.typelevel" %%% "jawn-ast" % JawnVersion % Test,
+      "org.scalameta" %%% "munit" % MunitVersion % Test,
+      "org.typelevel" %%% "munit-cats-effect-3" % MunitCatsEffectVersion % Test
+    ),
+    versionIntroduced := Map(
+      "3.0.0-M1" -> "1.0.1",
```

Comment:
Unrelated but this key can be dropped completely I believe.

Reply:
What was it for?

Reply:
sbt-spiewak used (uses?) it to determine what mimaPreviousArtifacts should be set to for each crossScalaVersion. We definitely don't need the M1/RC entries as we no longer compile for those, and the fact that 3.0.0 isn't here tells me it's no longer necessary.

Reply:
Gotcha. I can drop that.